src/packages/file-storage/drivers/local.driver.ts

```diff
@@ -67,8 +67,8 @@ export class LocalDriver implements StorageDriver {
     const domainStats = await Promise.all(
       dirents.map(async (dirent) => {
         const { name, path: domainPath } = dirent;
-        const createdAt = (await statAsync(join(domainPath, name))).birthtime;
-        return { name, createdAt };
+        const changedAt = (await statAsync(join(domainPath, name))).ctime;
+        return { name, changedAt };
```

Comment:
I observed that all domains were consistently updating locally. Upon closer examination, I found that while the files within the directories were being updated, the directories themselves were not. As a result, directories were only created once. Consequently, I needed to inspect the metadata to determine when any content inside the directory was updated (using ctime).